auto_nag/scripts/severity_inconsistency.py

```diff
@@ -0,0 +1,92 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+WHITEBOARD_PAT = re.compile(r"\][,; ]*?\[")
+
+
+class SeverityInconsistency(BzCleaner):
+    def description(self):
+        return "Bugs with inconsistent severity flags"
+
+    def has_needinfo(self):
+        return True
+
+    def handle_bug(self, bug, data):
+        flags = WHITEBOARD_PAT.split(bug["whiteboard"][1:-1])
+        whiteboard_severities = [flag for flag in flags if flag.startswith("access-s")]
```

Comment:
Did you see some bugs with 2 severities ?
If yes we could have a new tool to check this inconsistency.

Reply:
No, I didn't see bugs with more than one accessibility severity. At least in the current query result (the script should raise an exception in such case).

However, Whiteboard flags are separated in an inconsistent way. Examples:
- `[flag1] [flag2]`: just space
- `[flag1], [flag2]`: comma and space
- `[flag1][flag2]`: nothing

That's why I had to use regexp split instead of the string split method.